lib/ansible/plugins/callback/__init__.py

```diff
@@ -174,6 +174,9 @@ def set_option(self, k, v):
     def get_option(self, k):
         return self._plugin_options[k]
 
+    def get_options(self):
+        return self._plugin_options
+
```

Comment:
this should not be required as it uses `get_option` internally in the base class

Reply:
so we really were only missing `has_option`